custom/benin/tasks.py

```diff
@@ -0,0 +1,8 @@
+from corehq.apps.celery import task
+
+
+@task(queue='background_queue', ignore_result=True)
```

Comment:
Can we move the queue name to an environment settings since we want to run in a different queue eventually?

Reply:
Good catch, updated in [1ca177a](https://github.com/dimagi/commcare-hq/pull/34588/commits/1ca177a70882ac70c2956462dbb3160ebeb754d3)